src/v2/client/transactionMeta.js

```diff
@@ -95,4 +95,10 @@ export function isTransactionMetaValid(settings, meta) {
   }
 
   return true;
-}
\ No newline at end of file
+}
+
+export function useInteractionCodeFlow(settings) {
+  const authClient = settings.getAuthClient();
+  const transactionMeta = authClient.transactionManager.load();
```

Comment:
Can we reuse the [`getTransactionMeta`](https://github.com/okta/okta-signin-widget/blob/c07ce5a87879a4bf98faf40cf8239eda8f4d86e9/src/v2/client/transactionMeta.js#L21) method here?

Reply:
Yes.. but I think it's better to not use `getTransactionMeta` because it's async and will create new transaction meta if one does not exist